test/ae_database/AESetup.inc

```diff
@@ -0,0 +1,251 @@
+                                [encDetBit] [bit]
+                                $encDet
+                                [encRandBit] [bit] 
+                                $encRand";
+                break;
+            case "decimal":
+                $sql_create .= "[normDecimal] [decimal] (18, 5) NULL,
+                                [encDetDecimal] [decimal] (18, 5)
+                                $encDet
+                                [encRandDecimal] [decimal] (18, 5)
+                                $encRand";
+                break;
+            case "numeric":
+                $sql_create .= "[normNumeric] [numeric] (18, 5) NULL,
+                                [encDetNumeric] [numeric] (18, 5)
+                                $encDet
+                                [encRandNumeric] [numeric] (18, 5)
+                                $encRand";
+                break;
+            case "money":
+                $sql_create .= "[normMoney] [money] NULL,
+                                [encDetMoney] [money]
+                                $encDet
+                                [encRandMoney] [money] 
+                                $encRand";
+                break;
+            case "smallmoney":
+                $sql_create .= "[normSmallmoney] [smallmoney] NULL,
+                                [encDetSmallmoney] [smallmoney]
+                                $encDet
+                                [encRandSmallmoney] [smallmoney] 
+                                $encRand";
+                break;
+            case "binary":
+                $sql_create .= "[normBinary] [binary] (10) NULL,
+                                [encDetBinary] [binary] (10)
+                                $encDet
+                                [encRandBinary] [binary] (10)
+                                $encRand";
+                break;
+            case "varbinary":
+                $sql_create .= "[normVarbinary] [varbinary] (50) NULL,
+                                [encDetVarbinary] [varbinary] (50)
+                                $encDet
+                                [encRandVarbinary] [varbinary] (50)
+                                $encRand";
+                break;
+            case "varbinarymax":
+                $sql_create .= "[normVarbinarymax] [varbinary] (max) NULL,
+                                [encDetVarbinarymax] [varbinary] (max)
+                                $encDet
+                                [encRandVarbinarymax] [varbinary] (max)
+                                $encRand";
+                break;
+        }
+    }
+    
+    $sql_create .= " PRIMARY KEY CLUSTERED ([Id] ASC) ON [PRIMARY]) ";
+    
+    $stmt = sqlsrv_query($conn, $sql_create);
+    sqlsrv_assert($stmt === false, "Error in creating table.\n");
+}
+
+function CreateAETable($conn){
+    CreateAETableTypes($conn, "nvarchar", "int", "float", "datetime2", "bit");
+}
+
+function PopulateAETable($conn){
```

Comment:
Why do we need CreateAETable() and PopulateAETable() here? Are they a mix of column types?